.changeset/fresh-chairs-wait.md

```diff
@@ -1,7 +1,7 @@
 ---
-"@fluidframework/common-utils": major
+"@fluidframework/common-utils": minor
```

Comment:
I see quite a lot of major -> minor switches. Can you please comment for clarity, in terms of upcoming rc release and follow up ones. Do we need tools update?

Reply:
All changes for main should be minor and that's what the tools do. But people can create the files manually which is what I think happened here.